Analysis/include/AnaTuple.h

```diff
@@ -109,7 +109,7 @@ namespace analysis {
     VAR_LIST(float, npv, HT_total, HT_otherjets, lhe_HT, n_jets, n_jets_eta24, n_jets_eta24_eta5, \
                     n_selected_gen_jets, n_selected_gen_bjets, genJets_nTotal, \
                     jets_nTotal_hadronFlavour_b, jets_nTotal_hadronFlavour_c) /* other global event quantities */ \
-    WVAR_LIST(_, btag_Loose, btag_Medium, btag_Tight, btag_IterativeFit) \
+    WVAR_LIST(_, btag_Loose, btag_Medium, btag_Tight, btag_IterativeFit, btag_IterativeFit_jes) \
```

Comment:
WVAR_LIST for btag_IterativeFit and btag_IterativeFit_jes doesn't seem to be adequate any longer. Only two branches should be defined:
```c++
VAR_LIST(float, weight_btag_IterativeFit, weight_btag_IterativeFit_withJES) \
```